urdf_model/include/urdf_model/utils.h

```diff
@@ -0,0 +1,67 @@
+/*********************************************************************
+* Software License Agreement (BSD License)
+*
+*  Copyright (c) 2008, Willow Garage, Inc.
```

Comment:
New files get a new copyright(owner), by the author with the current date. It's typically recommended to use the same license for consistency if allowed by the owner. 

Any other file with "significant" changes should have the copyright owner list extended with the modification years.